app/assets/javascripts/mirador_messager.js

```diff
@@ -0,0 +1,17 @@
+var miradorInstance;
+$(document).on('ready turbolinks:load', () => {
+  $('[data-miradorfromscratch]').each(function(i, value) {
+    
+    // Setup Mirador to use config already existing in parent form
+    parent.$('#mirador-modal').on('shown.bs.modal', function () {
+      var miradorConfig = JSON.parse(parent.$('[name="mirador_config"]').val());
```

Comment:
This actually gives the same configuration to all viewers on the page.

Reply:
@aeschylus mind creating a spin off ticket so we can come back to this?